src/main/resources/jenkins/plugins/logstash/LogstashInstallation/global.jelly

```diff
@@ -22,6 +22,10 @@
         checkUrl="'${resURL}/descriptorByName/LogstashInstallation/checkString?value='+escape(this.value)" />
     </f:entry>
     <f:advanced>
+      <f:entry title="${%MIME type}" field="mimeType">
+      <f:textbox value="${descriptor.mimeType}" default="application/json"
```

Comment:
I'm not sure if the existing installations will get null or the default value.
Can we set the default to match the previous value?

Reply:
I am not sure if I understand that correctly. Even in case of `null`, Content-Type will be set to `ContentType.APPLICATION_JSON`
Default value is no different then the previous one it is just that it is set in diff way :)
i.e. The default value is set to `application/json` and the String entity is initialized with `charset=UTF-8`.

Reply:
OK I get it now.
Can we make it more explicit in the description that the mimeType input doesn't accept any parameters (incl. charset)?